exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -90,12 +89,13 @@ def _handle_output(self, output_generator, image_info: ImageInfo):
                 still_running_logger.log()
                 log_handler.handle_log_lines(log_line)
 
-    def _enable_ssh_access(self, container: Container):
-        sshkey = SshKey.from_cache()
-        copy = DockerContainerCopy(container)
-        content = sshkey.public_key_as_string("itde-ssh-access")
-        copy.add_string_to_file(".ssh/authorized_keys", content)
-        copy.copy("/root/")
```

Comment:
we still need this, in case of a c4 based docker images

Reply:
OK, will be re-added in next push